### DISCUSSION TITLE
How to embed additional resources files according to platforms?

### DISCUSSION THREAD

**Discussion by boan-anbo** [tauri-apps/tauri#9224] [Q&A] [answered]
Let's say I have separate external library binaries for different platforms. MacOS will not need the Win library. 

How do I include in the Tauri installation build only the libraries for the targeted platform?

I can't tell from the docs: https://tauri.app/v1/guides/building/resources/

Thanks

**Comment by FabianLars** [comment, marked as answer]
You can have platform specific configs: <https://tauri.app/v1/api/config#platform-specific-configuration>

**Reply by boan-anbo** [reply]
Awesome, thank you!

**Comment by haexhub** [comment]
Hi.
How I'm supposed to include binaries depending on CPU in tauri v2? I've a sqlite database with an extension. But depending on CPU I need a different binary.

**Reply by FabianLars** [reply]
until https://github.com/tauri-apps/tauri/issues/8501 is resolved you'll need to use the --config flag instead of relying on tauri's platform specific config files.

**Reply by haexhub** [reply]
thanks for your answer, I'll try that out. I also asked this question in the issue you mentioned, because I didn't saw the answer here.

**Reply by FabianLars** [reply]
yeah i only mentioned it briefly. you'll need to use the `--config` flag to select a arch specific config yourself. so for example you have a `tauri.windows-aarch64.conf.json` file and then select it yourself when you build for that target
```
npm run tauri build -- --target aarch64-pc-windows-msvc --config path/to/tauri.windows-aarch64.conf.json
```
